packages/graphback-runtime-knex/src/PostgreDBDataProvider.ts

```diff
@@ -14,14 +14,14 @@ import { GraphbackDataProvider, GraphbackPage, NoDataError, AdvancedFilter } fro
  * NOTE: For Postgres use dedicated `PgKnexDBDataProvider` that implements more performant creation method.
  */
 //tslint:disable-next-line: no-any
-export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
+export class PostgreDBDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
```

Comment:
Copy this implementation into separate file. 
KnexDataProvider should not be changed

Reply:
oki